internal/configs/version1/nginx-plus.ingress.tmpl

```diff
@@ -55,6 +55,14 @@ server {
 
 	{{if $server.SSL}}
 	{{if not $server.GRPCOnly}}
+	{{- if $server.SSLRedirect}}
+	if ($scheme = http) {
+		return 301 https://$host:{{index $server.SSLPorts 0}}$request_uri;
+	}
+	{{- end}}
+	{{end}}
+	{{- end}}
+
 	{{- if $server.HSTS}}
```

Comment:
with this suggestion HSTS will only be enabled when TLS termination is enabled and `hsts-behind-proxy` annotation/configmapkey is true.
```suggestion
	{{- if and $server.HSTS (or $server.SSL $server.HSTSBehindProxy)}}
```

Reply:
Additionally, please move lines 66-78 (the `{{- if and $server.HSTS (or $server.SSL $server.HSTSBehindProxy)}} ... {{end}}` block) above line 56 (`{{if $server.SSL}})`. This will fix the warning message like this `*84 using uninitialized "hsts_header_val" variable`.

Reply:
@pleshakov Thanks for taking the time to review the change! The suggested changes would apply for nginx-plus and regular nginx right ? Just making sure!

Reply:
yep. for both templates! thanks for double checking.

Reply:
Pushed suggested changes!